sdk/edgezones/arm-edgezones/README.md

```diff
@@ -0,0 +1,109 @@
+# Azure MicrosoftEdgeZones client library for JavaScript
+
+This package contains an isomorphic SDK (runs both in Node.js and in browsers) for Azure MicrosoftEdgeZones client.
+
+
+
+[Source code](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/edgezones/arm-edgezones) |
+[Package (NPM)](https://www.npmjs.com/package/@azure/arm-edgezones) |
+[API reference documentation](https://docs.microsoft.com/javascript/api/@azure/arm-edgezones?view=azure-node-preview) |
+[Samples](https://github.com/Azure-Samples/azure-samples-js-management)
+
+## Getting started
+
+### Currently supported environments
+
+- [LTS versions of Node.js](https://github.com/nodejs/release#release-schedule)
+- Latest versions of Safari, Chrome, Edge and Firefox.
+
+See our [support policy](https://github.com/Azure/azure-sdk-for-js/blob/main/SUPPORT.md) for more details.
+
+### Prerequisites
+
+- An [Azure subscription][azure_sub].
+
+### Install the `@azure/arm-edgezones` package
+
+Install the Azure MicrosoftEdgeZones client library for JavaScript with `npm`:
+
+```bash
+npm install @azure/arm-edgezones
+```
+
+### Create and authenticate a `MicrosoftEdgeZones`
```

Comment:
why `MicrosoftEdgeZones`?

Reply:
Can we use `@clientName` to rename it, also title in the config https://github.com/Azure/autorest.typescript/issues/2355